application/jira/v0/config/tasks.json

```diff
@@ -0,0 +1,744 @@
+{
+  "$defs": {
+    "common-query-params": {
+      "start-at": {
+        "default": 0,
+        "description": "The starting index of the returned boards. Base index: 0. Default is 0",
+        "instillUIOrder": 3,
+        "title": "Start At",
+        "instillFormat": "integer",
+        "instillAcceptFormats": [
+          "integer"
+        ],
+        "instillUpstreamTypes": [
+          "value",
+          "reference",
+          "template"
+        ],
+        "type": "integer"
+      },
+      "max-results": {
+        "default": 50,
+        "description": "The maximum number of boards to return. Default is 50",
+        "instillUIOrder": 4,
+        "title": "Max Results",
+        "instillFormat": "integer",
+        "instillAcceptFormats": [
+          "integer"
+        ],
+        "instillUpstreamTypes": [
+          "value",
+          "reference",
+          "template"
+        ],
+        "type": "integer"
+      },
+      "update-history": {
+        "description": "Whether the project in which the issue is created is added to the user's Recently viewed project list, as shown under Projects in Jira.",
+        "title": "Update History",
+        "instillUIOrder": 5,
+        "instillFormat": "boolean",
+        "instillAcceptFormats": [
+          "boolean"
+        ],
+        "instillUpstreamTypes": [
+          "value"
+        ],
+        "type": "boolean"
+      }
+    },
+    "issue": {
+      "properties": {
+        "id": {
+          "description": "The ID of the issue",
+          "instillUIOrder": 0,
+          "title": "ID",
+          "instillFormat": "string",
+          "type": "string"
+        },
+        "key": {
+          "description": "The key of the issue",
+          "instillUIOrder": 1,
+          "instillFormat": "string",
+          "title": "Key",
+          "type": "string"
+        },
+        "self": {
+          "description": "The URL of the issue",
+          "instillUIOrder": 2,
+          "instillFormat": "string",
+          "title": "Self",
+          "type": "string"
+        },
+        "fields": {
+          "description": "The fields of the issue. All navigable and Agile fields are returned",
+          "instillUIOrder": 3,
+          "instillFormat": "object",
+          "title": "Fields",
+          "type": "object",
+          "required": []
+        },
```

Comment:
Since there are plenty of unknown and user-defined fields in `fields`, I left it blank. I just learned we can use json or semi-json as types. I can change the type and make it possible for users to access the information in this field.